### DISCUSSION TITLE
Question about Q-Date Vs Input with Date Mask

### DISCUSSION THREAD

**Discussion by CynthiaBlue** [quasarframework/quasar#11525] [General - Components / Directives / etc]
I'm not sure if I should use a q-date component or just a q-input with a date mask.

I'd love advice.

I currently am taking a date with a q-input and I need to populate other fields based on that first date.  I am pretty sure I'm going to have to do more date manipulation in the future, like calculating the number of days between dates, etc.

What do you use and why? Thanks.

**Comment by TobyMosque** [comment]
u can create a component who wrappers a QInput, QPopupProxy and the QDate.
u just need to keep your value in a standardized format, quasar prefers `yyyy/MM/dd HH:mm:ss`, but u can use the ISO (`yyyy-MM-ddTHH:mm:ss`) too, but not all browsers (I'm looking to you Safari) handles ISO dates in the same way.:

here a example: (format MM/dd/yyyy)
https://codepen.io/tobias-mesquita/pen/zYExYYY?editors=1010
